packages/regex/src/index.ts

```diff
@@ -36,6 +36,7 @@ export const sixDigitsCode = /^[0-9]{6}$/
 export const url =
   /^http(s)?:\/\/?[\w.-]+(?:\.[\w.-]+)+[\w\-._~:/?#[\]@!$&'()*+,;=.]+$/
 export const hexadecimal = /^[0-9a-fA-F]+$/
+export const s3BucketName = /^[a-z0-9._-]{3,}$/
```

Comment:
Still needed with the new hive backend ? 🤔

Reply:
It's part of the S3 spec, so yes